### PR TITLE
feat: new collection design

### LINKED ISSUE #5392: Turn on new collection design by default
We can switch to the new collection design as default, we won't plan to push it yet out as I sense some polishing will be ongoing.

Reserved for @kodadot/internal-dev someone to pick it up

Issues being done in parallel
- https://github.com/kodadot/nft-gallery/issues/5334
- https://github.com/kodadot/nft-gallery/issues/4911
- https://github.com/kodadot/nft-gallery/issues/5286
- https://github.com/kodadot/nft-gallery/issues/5388
- https://github.com/kodadot/nft-gallery/issues/5374
- https://github.com/kodadot/nft-gallery/pull/5345
- https://github.com/kodadot/nft-gallery/issues/5272
- https://github.com/kodadot/nft-gallery/issues/5000

### PR DESCRIPTION
**Pull request by preschian**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Context

- [x] Closes #5392
- [x] added meta opengraph for collection

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=DY4SQF2iD456tH89aQtz5wv1EV3BbSW8wKKuMcwbmXaj1pM)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

**Review by roiLeo — COMMENTED** [review]
When I go from "Activity" Tab to "Items" Tab, I still got `?redesign=true` in query

Screenshot 2023-03-29 at 09-59-16 kittys KodaDot - NFT Market Explorer

- Report NFT should be disabled for now
- Same goes for download?

maybe should be done in #5334 

Note: issue with seo on deploy-preview?
Screenshot 2023-03-29 at 10-10-07 Sharing Debugger - Meta for Developers

**Comment by yangwao** [comment]
I would like to do a very update to production before we enable this :)

**Comment by yangwao** [comment]
mergeable?

File: tests/cypress/e2e/basicE2E.cy.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ describe('landingBsx', () => {
   })
 })
 
-describe('snekCollectionItem', () => {
+describe.skip('snekCollectionItem', () => {
```
Comment (vikiival) [inline tests/cypress/e2e/basicE2E.cy.ts:18]: why skip?

Comment (yangwao) [reply tests/cypress/e2e/basicE2E.cy.ts:18]: is not working 😅

**Review by vikiival — APPROVED** [review]

**Comment by yangwao** [comment]
pay 50 usd

**Comment by yangwao** [comment]
> Note: issue with seo on deploy-preview?

telegram
image

but yeah twitter doesn't work 

oh also card dev validator for twitter is dead 
- https://twittercommunity.com/t/card-validator-preview-removal/175006

**Pull request merged**